packages/jaeger-ui/src/components/DeepDependencyGraph/Header/tmp-data.tsx

```diff
@@ -0,0 +1,841 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// These strings are lifted from
+// https://github.com/moby/moby/blob/e105a74c54193e0ebdeddd93afc8746c33aeca73/pkg/namesgenerator/names-generator.go
+// istanbul ignore next
+export default [
```

Comment:
Please include `'focalService'` and `'focalOperation'` so that the mock path data can be used for other features while we wait on the BE to provide services and operations.

Reply:
I was thinking this data would be removed when we have this wired up. For services, in the interim, we can use the search API.